scripts/clean-up-localizations.js

```diff
@@ -0,0 +1,65 @@
+const fs = require('fs');
+const glob = require('glob');
+
+const localizationFile_en = 'src/assets/i18n/en_US.json';
+const localizationFile_fr = 'src/assets/i18n/fr_FR.json';
+const localizationFile_de = 'src/assets/i18n/de_DE.json';
+
+// regular expressions for patterns of not explicit used localization keys (dynamic created keys, error keys from rest calls)
+// new patterns have to be added here
+const patterns = ['account.login..*.message', '.*.?error..*'];
+const regEx = new RegExp(patterns.join('|'), 'i');
```

Comment:
`const regEx = /account\.login\..*\.message|.*\.error\..*/i`

If a dot is unescaped, it matches any character. In this case we want to explicitly match dots.